dev/com.ibm.ws.cdi.mp.context/resources/com/ibm/ws/cdi/mp/context/resources/CDI_MP_CONTEXT.nlsprops

```diff
@@ -0,0 +1,33 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.concurrent/resources/com/ibm/ws/concurrent/resources/CWWKCMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKC
+#COMPONENTNAMEFOR CWWKC MicroProfile Context Propagation
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
```

Comment:
Change
'Strings in this file, which do NOT contain replacement variables are NOT processed by the MessageFormat class'
to
'Strings in this file, which do NOT contain replacement variables are NOT processed by the MessageFormat class (single quotation mark must be coded as one single quote '). ' 
(single quote must be coded as one single quote ').  '

Reply:
this is a boilerplate header comment (not an NLS message) so I'm going to leave as-is. Any line that starts with `#` in a .nlsprops file is a comment and a user will never see it

Reply:
Okay.